installer/pom.xml

```diff
@@ -76,9 +74,11 @@ limitations under the License.
                         <configuration>
                             <shadedArtifactAttached>true</shadedArtifactAttached>
                             <transformers>
-                                <transformer implementation=
-                                                     "org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
                                     <mainClass>com.amazon.aws.partners.saasfactory.saasboost.SaaSBoostInstall</mainClass>
+                                    <manifestEntries>
+                                        <Multi-Release>true</Multi-Release>
+                                    </manifestEntries>
```

Comment:
It gets rid of this scary sounding warning that prints to STDOUT when you execute the shaded JAR. It's an issue with the shade maven plugin.
WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.